File: app/src/test/java/com/example/projectimplimentation/TodoResponseTest.java
package com.example.projectimplimentation;

import com.example.projectimplimentation.network.TodoResponse;

import org.junit.Test;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertFalse;

public class TodoResponseTest {

    @Test
    public void todoResponse_holdsServerFields() {
        TodoResponse response = new TodoResponse(1, "delectus aut autem", false);

        assertEquals(1, response.getId());
        assertEquals("delectus aut autem", response.getTitle());
        assertFalse(response.isCompleted());
    }
}

File: app/src/test/java/com/example/projectimplimentation/TodoResponseTest.kt
package com.example.projectimplimentation

import com.example.projectimplimentation.network.TodoResponse
import org.junit.Assert.assertEquals
import org.junit.Assert.assertFalse
import org.junit.Test

class TodoResponseTest {

    @Test
    fun todoResponse_holdsServerFields() {
        val response = TodoResponse(id = 1, title = "delectus aut autem", completed = false)

        assertEquals(1, response.id)
        assertEquals("delectus aut autem", response.title)
        assertFalse(response.completed)
    }
}
